spec/requests/additional_content_boxes_spec.rb

```diff
@@ -1,35 +1,35 @@
 require "rails_helper"
 
 RSpec.describe "AdditionalContentBoxes", type: :request do
-  let(:tag) { create(:tag) }
-  let(:user) { create(:user) }
-  let(:regular_article) { create(:article, user: user, tags: [tag.name]) }
+  let_it_be(:tag) { create(:tag) }
+  let_it_be(:user) { create(:user) }
+  let_it_be(:regular_article) { create(:article, user: user, tags: [tag.name]) }
 
   describe "GET /additional_content_boxes" do
     it "returns an article if there is a published/featured one" do
-      suggestion = build_stubbed(:article, published: true, featured: true, path: "blah")
-      suggestion.title = "Title of the article"
-
-      fake = instance_double(Suggester::Articles::Classic)
-      allow(Suggester::Articles::Classic).to receive(:new).and_return(fake)
-      allow(fake).to receive(:get).and_return([suggestion])
```

Comment:
because of this mock we weren't actually testing the `get` method, the advantage of building an article that will be found by the "suggester" is that we're sure that the test is actually doing something useful